content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -47,6 +47,13 @@ turn, notify the customer that the order (or a part of it) is on its way.
    module in order to provide the tracking reference. There is unfortunately no workaround to comply
    with this new Amazon policy.
 
+.. tip::
+   - If your chosen carrier isn't one supported by Odoo, you can still create a carrier bearing its
+     name (e.g. create a carrier named `Colissimo`). This name is case insensitive, but be careful
+     about any typo, as Amazon wouldn't recognize it.
```

Comment:
Small change to this sentence to make it present tense and sound a little smoother:

`about typos, as Amazon won't recognize them.`